types/globby/index.d.ts

```diff
@@ -3,19 +3,20 @@
 // Definitions by: Douglas Duteil <https://github.com/douglasduteil>
 //                 Ika <https://github.com/ikatyang>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
 
-import { IOptions } from 'glob';
+import { Options } from 'fast-glob';
 
 /**
  * Returns a `Promise<Array>` of matching paths.
  */
-declare function globby(patterns: string | string[], options?: IOptions): Promise<string[]>;
+declare function globby(patterns: string | string[], options?: Options): Promise<string[]>;
 
 declare namespace globby {
     /**
      * Returns an `Array` of matching paths.
      */
-    function sync(patterns: string | string[], options?: IOptions): string[];
+    function sync(patterns: string | string[], options?: Options): string[];
     /**
      * Returns an `Array<Object>` in the format `{ pattern: string, opts: Object }`,
      * which can be passed as arguments to [node-glob](https://github.com/isaacs/node-glob).
```

Comment:
These looks outdated, we should also update the docs, just need to copy-paste from the [API](https://github.com/sindresorhus/globby#api).